lib/ansible/module_utils/mysql.py

```diff
@@ -69,8 +71,8 @@ def mysql_connect(module, login_user=None, login_password=None, config_file='',
     if connect_timeout is not None:
         config['connect_timeout'] = connect_timeout
 
-    db_connection = MySQLdb.connect(**config)
+    db_connection = mysql_driver.connect(**config)
```

Comment:
Uhh, will this not explode if `mysql_driver` is `None`? Could we make an assertion here? I assume this function is only called internally by the module and therefore we can give any developer a hard stop if they're missing the dependency.